docs/guides/src/main/server/caching.adoc

```diff
@@ -10,52 +10,110 @@ includedOptions="cache cache-*">
 Keycloak is designed for high availability and multi-node clustered setups.
 The current distributed cache implementation is built on top of https://infinispan.org[Infinispan], a high-performance, distributable in-memory data grid.
 
-All available cache options are build options, so they need to be applied to a `build` of Keycloak before starting.
-
-== Enable distributed caching
+== Enable clustering
 When you start Keycloak in production mode, by using the `start` command, caching is enabled and all Keycloak nodes in your network are discovered.
+By default, caches are using a `UDP` transport stack so that nodes are discovered using IP multicast transport based on UDP. While not suitable
```

Comment:
> By default, caches are using a `UDP` transport stack so that nodes are discovered using IP multicast transport based on UDP. While not suitable
> for most production deployments, the server allows you to choose other transport stacks as you will see later in this guide.

This is a bit ambiguous. Do you mean 
1) UDP IP Multicast is not suitable for most production deployments? 
2) other transport stacks are not suitable for most production deployments?

Suggestion (assuming 1. is what is wanted): 

By default, caches are using a `UDP` transport stack so that nodes are discovered using IP multicast transport based on UDP. For most production environments, there are better alternatives to UDP for node discovery available. Keycloak allows you to either choose from a set of pre-defined transport stacks or to define your own custom stack, as you will see later in this guide.

Reply:
But you are basically saying the same thing using different words, no?

UDP is not suitable for most deployments and we provide different alteratives. That is basically it.

Reply:
as stated before, it is ambiguous for me to read, so can mean 2 things, either:
* "UDP" is not suitable for prod envs, so the "while" refers to the sentence before. 
or:
*  "cache stacks" are not suitable for production envs, that happens when you read the second sentence alone: "While not suitable for most production deployments, the server allows you to choose other transport stacks as you will see later in this guide."

the "while" is ambiguous, can reference 2 things, so I changed it in my suggestion to make it more clear that UDP is meant.

 But not hard on it, should be clear from context. go with whatever you think is best.

Reply:
I see now. I can change that, better is to avoid confusion.